### PR TITLE
Merge docs branch into dev-master

### PR DESCRIPTION
**Pull request by grabear**
This is a pretty large change.  But git checkout this site and then `pkgdown::preview_site()`..

# MicrobiomeR 0.2.0

## Documentations

* Added vignettes to package and pkgdown config.
  * About
  * Introduction
  * Data Wrangling
  * Filtering
  * Analysis
* Added reference sections to pkgdown config corresponding to @family tag.
  * Import
  * Formatting
  * Validation
  * Filtering (Basic, Advanced, Other)
  * Visualization
  * Color Palettes

## Functional Changes

* Renamed `sample_filter` to `sample_id_filter`.
* Renamed `get_phyloseq_obj` to `create_phyloseq`.
* Renamed `format_metacoder_object` to `as_custom_format`.
* Updated `vlookup` and `get_color_palette` parameters.
* Added the `root_phyloseq_tree` function.
* Fixed minor bugs in code and docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,28 +3,39 @@
...
-    - Metacoder for data analysis.
-    - Ggplot2 for data visualization.
-- Publication ready figures for various ranks.
-    - Metacoder Heat Trees
-    - MicrobiomeR Correlation Plots
-    - MicrobiomeR Stacked Bar Charts
-- Easily access taxonomic ranks from data objects.
-- Import various data types from different pipelines.
+An R package for microbiome analysis that incorporates [phyloseq](https://github.com/joey711/phyloseq), 
+[metacoder](https://github.com/grunwaldlab/metacoder), [taxa](https://github.com/ropensci/taxa), and [microbiome](https://github.com/microbiome/microbiome/) into a fully featured bioinformatics workflow.
```
Comment (sdhutchins) [inline README.md]:
```suggestion
[metacoder](https://github.com/grunwaldlab/metacoder), [taxa](https://github.com/ropensci/taxa), and [microbiome](https://github.com/microbiome/microbiome/) in order to standardize and simplify common microbiome analysis methods and workflows.
```

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,30 @@
+# MicrobiomeR 0.2.0
+
+## Documentations
```
Comment (sdhutchins) [inline NEWS.md]:
```suggestion
## Documentation Updates
```
Idk here. Go with whatever you're comfortable with. :smile:

File: R/phyloseq.R [hunk 3]
```diff
@@ -47,50 +50,110 @@
...
+  }
+}
+
+#' @title Root phyloseq tree.
+#' @description A function for rooting and saving a phylogenetic tree.
+#' @param phyloseq_object A phyloseq object that contains a phylogenetic tree.
+#' @param tree_path The path to the existing or desired phylogenetic tree file.
+#' @param save_rooted_tree A logical that determines weather or not the rooted tree
+#' is saved.
+#' @param recursive If the directory doesn't exists create the parent directories
```
Comment (sdhutchins) [inline R/phyloseq.R]:
```suggestion
#' @param recursive If the directory doesn't exist, create the parent directories
```

File: R/phyloseq.R [hunk 4]
```diff
@@ -47,50 +50,110 @@
...
+#' that don't exist as well, Default: TRUE
+#' @return Returns a phyloseq object with a rooted tree.
+#' @pretty_print TRUE
+#' @details This function is a helper function to get a proper phyloseq object for
+#' downstream analysis.  Some analysis requre a rooted tree.  The function saves
```
Comment (sdhutchins) [inline R/phyloseq.R]:
```suggestion
#' downstream analysis.  Some analyses require a rooted tree.  The function saves
```

**Comment by grabear** [comment]
Hooray!!  @sdhutchins

**Comment by sdhutchins** [comment]
Now...REST! lol @grabear (I know you won't. 😆 )

**Pull request merged**